### PR TITLE
nmap: add German translation

### PR DESCRIPTION
**Pull request by Gareck-OG**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/common/nmap.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# nmap
+
+> Netzwerk erkundungs Werkzeug und Security / Port Scanner.
+> Manche Funktionen können nur benutzt werden bei ausführung von Nmap mit Root Rechten.
+> Weitere Informationen: <https://nmap.org>.
+
+- Überprüft ob eine IP-Adresse Online ist und Sannt nach dem Betriebssystem:
```
Comment (pixelcmtd) [inline pages.de/common/nmap.md:7]:
```suggestion
- Überprüfe ob eine IP-Adresse online ist und versuche, das Betriebssystem herauszufinden:
```

File: pages.de/common/nmap.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+`nmap -O {{ip_oder_hostname}}`
+
+- Überprüft nur ob ein Host Online ist (Ping Scan):
```
Comment (pixelcmtd) [inline pages.de/common/nmap.md:11]:
```suggestion
- Überprüfe nur ob die angegebenen Hosts online sind (Ping Scan) und ihre Domain-Namen:
```

File: pages.de/common/nmap.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+`nmap -sn {{ip_oder_hostname}} {{optional_noch_eine_addresse}}`
+
+- Scanne zusätzlich mit aggressiver Betriebssystem erkennung, Version erkennung, nach Scripts und nach Traceroute:
```
Comment (pixelcmtd) [inline pages.de/common/nmap.md:15]:
```suggestion
- Scanne zusätzlich mit Skripten, Service-Erkennung, Betriebssystem-Fingerprinting und Traceroute:
```

File: pages.de/common/nmap.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+`nmap -A {{addresse_oder_addressen}}`
+
+- Scanne eine spezifische Liste an Ports (benutze '-p-' für alle Ports von 1 bis 65535):
+
+`nmap -p {{port1,port2,...,portN}} {{addresse_oder_addressen}}`
+
+- Führe Dienst und Version Erkennung auf den top 1000 Ports mit den Standard NSE Scripts aus; Speicher das Ergebins ('-oN') in der Ausgabe Datei:
```
Comment (pixelcmtd) [inline pages.de/common/nmap.md:23]:
```suggestion
- Führe Dienst- und Versions-Erkennung auf den top 1000 Ports mit den Standard NSE Skripten aus; und schreibe das Ergebnis ('-oN') in der Ausgabe Datei:
```

File: pages.de/common/nmap.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+`nmap -sC -sV -oN {{ergebnis.txt}} {{addresse_oder_addressen}}`
+
+- Scanne Ziel(e) vorsichtig mit 'default and safe' NSE Scripts:
+
+`nmap --script "default and safe" {{addresse_oder_addressen}}`
+
+- Scanne einen Web-Server, der auf den Standard Ports 80 und 443 mit allen verfügbaren 'http-*' NSE Scripts:
```
Comment (pixelcmtd) [inline pages.de/common/nmap.md:31]:
```suggestion
- Scanne einen Web-Server, der auf den Standard Ports 80 und 443 läuft, mit allen verfügbaren 'http-*' NSE Scripts:
```

File: pages.de/common/nmap.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+`nmap --script "http-*" {{addresse_oder_addressen}} -p 80,443`
+
+- Führe einen sehr langsamen verborgenen Scan ('-T0') aus um die Entdeckung von IDS/IPS zu umgehen und benutze Köder IP Adressen ('-D'):
```
Comment (pixelcmtd) [inline pages.de/common/nmap.md:35]:
```suggestion
- Führe einen sehr langsamen verborgenen Scan ('-T0') aus um die Entdeckung von IDS/IPS zu umgehen und benutze Köder IP-Adressen ('-D'):
```

File: pages.de/common/nmap.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
+# nmap
+
+> Netzwerk erkundungs Werkzeug und Security / Port Scanner.
```
Comment (pixelcmtd) [inline pages.de/common/nmap.md:3]:
```suggestion
> Netzwerk-Erkundungs-Werkzeug und Security / Port Scanner.
```

File: pages.de/common/nmap.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+> Manche Funktionen können nur benutzt werden bei ausführung von Nmap mit Root Rechten.
```
Comment (pixelcmtd) [inline pages.de/common/nmap.md:4]:
```suggestion
> Manche Funktionen können nur benutzt werden, wenn Nmap mit Root Rechten ausgeführt wird.
```

**Comment by Gareck-OG** [comment]
I am clearly not made for this......

**Review by pixelcmtd — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**